### PR TITLE
Initial addtion to master branch

### PR DESCRIPTION
**Pull request by deshankoswatte**
- SAMLInitializer which initializes OpenSAML3.
- UnmarshallUtils which contains common operations performed at the unmarshalling process

### CODE REVIEW

File: components/org.wso2.carbon.identity.saml.common.util/pom.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                                </rule>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.saml.common.util/pom.xml]: Add new line.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (madurangasiriwardena) [inline pom.xml]: Add new line.

File: components/org.wso2.carbon.identity.saml.common.util/pom.xml (hunk 1 above)
Comment (deshankoswatte) [reply components/org.wso2.carbon.identity.saml.common.util/pom.xml]: Fixed

File: pom.xml (hunk 2 above)
Comment (deshankoswatte) [reply pom.xml]: Fixed

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: java
+jdk:
+    - openjdk8
+cache:
+  directories:
+  - .autoconf
+  - $HOME/.m2
+  timeout: 600
+script: mvn clean install
```
Comment (deshankoswatte) [inline .travis.yml]: Add a new line

Comment (deshankoswatte) [reply .travis.yml]: Fixed

**Review by madurangasiriwardena — APPROVED** [review]

**Pull request merged**